source/PlayerInfo.cpp

```diff
@@ -2170,6 +2177,16 @@ bool PlayerInfo::SelectShips(const vector<const Ship *> &stack, bool hasShift)
 
 
 
+// void PlayerInfo::SelectShips(const std::set<int> &selectedSet)
```

Comment:
Did you mean to include these changes?

Reply:
Oh, too many branches and too little git practice in the last time ...
Thanks for pointing out, this was a mistake and is part of another branch but obviously slipped in. Should be fixed now, although by an ugly rebase.